src/services/location.js

```diff
@@ -128,11 +128,12 @@ ngeo.Location.prototype.getParam = function(key) {
 /**
  * Get a param from the fragment of the location's URI.
  * @param {string} key Param key.
- * @return {string} Param value.
+ * @return {string|undefined} Param value.
  * @export
  */
 ngeo.Location.prototype.getFragmentParam = function(key) {
-  return /** @type {string} */ (this.getFragmentUri_().getQueryData().get(key));
+  var val = /** @type {string} */ (this.getFragmentUri_().getQueryData().get(key));
+  return val !== undefined ? goog.string.urlDecode(val) : undefined;
```

Comment:
I added a new module `ngeo.string`. There are still other places where `goog.string` is used, but I don't want to handle it in this PR.